src/WebJobs.Script/Workers/Profiles/WorkerProfileManager.cs

```diff
@@ -97,7 +98,13 @@ public bool IsCorrectProfileLoaded(string workerRuntime)
             {
                 profileId = profile.ProfileId;
             }
-            return _activeProfile.Equals(profileId);
+
+            var activeProfile = string.Empty;
+            if (!_activeProfiles.TryGetValue(workerRuntime, out activeProfile))
+            {
+                activeProfile = string.Empty;
```

Comment:
it's already being set to an empty string on 102, we shouldn't need to do that again? You could probably not use the if statement here if `TryGetValue` returning false doesn't lead to a different action

Reply:
When TryGetValue returns false, activeProfile is set to null. So setting it to empty. I have corrected the initialization.